lib/api/template.coffee

```diff
@@ -35,7 +39,8 @@ exports.remove = (args) ->
 
 exports.list = (args) ->
   __track('api', { name: 'template-list' })
-  sprout.list(args)
+  _.keys(sprout.templates)
```

Comment:
this is returning an Array. I'm not sure if that's what you're meaning to do

![screen shot 2015-06-03 at 11 43 28 am](https://cloud.githubusercontent.com/assets/49507/7964506/159d7740-09e6-11e5-9fe1-648fe4d2622a.png)